x/metadata/types/msg.go

```diff
@@ -113,6 +113,9 @@ func (msg MsgWriteScopeRequest) ValidateBasic() error {
 	if err := msg.ConvertOptionalFields(); err != nil {
 		return err
 	}
+	if _, found := msg.Scope.GetOwnerIndexWithAddress(msg.Signers[0]); !found {
+		return fmt.Errorf("the first message signer %s is not an owner", msg.Signers[0])
+	}
```

Comment:
Also, this can't be here. It would cause most messages that use authz authorizations to fail.

Reply:
do we need this kind of checking at all given metadata should look up the address and see that it is a smart contract so that it gets forced against the PROVENANCE role? seemingly so long as the smart contract address is first, and the execute contract invoker is the second address on the message... the rest of the checks are the responsibility of the metadata module